index.js

```diff
@@ -254,14 +254,17 @@ ExtractTextPlugin.prototype.apply = function(compiler) {
 									compilation.errors.push(err);
```

Comment:
The actual problem is that `compilation.rebuildModule` does not call the callback with an `err` even if there was a compilation failure -- when the `module.meta[NS]` is unset bug is hit, `module.errors` is (AFAICT) always non-empty.